.rubocop.yml

```diff
@@ -65,3 +65,15 @@ Lint/MissingSuper:
 
 Naming/VariableNumber:
   Enabled: false
+
+Primer/DeprecatedArguments:
+  Enabled: true
+  Deprecated:
+    color:
+      blue: ":link"
+      gray_dark: ":primary"
+      gray: ":secondary"
+      gray_light: ":tertiary"
+      green: ":success"
+      yellow: ":warning"
+      red: ":danger"
```

Comment:
I thought this was a good place to put the arguments this way it can be configured by consumers.

Reply:
Is this something we want consumers to be able to configure? I'm just a bit concerned about requiring teams to copy/paste mappings into their configs. Seems like we might end up with a hodgepodge of correct and incorrect mappings in various repos 🤔

Reply:
I think most consumers would extend this config so they'll get what we have listed anyways, but will have the ability to change if it's taking them longer to migrate.

Reply:
Hmm ok I see what you mean. Sounds good to me 👍

Reply:
🤔 I have a similar hesitation to @camertron. My vote would be to hardcode these mappings inside the linter for now and to revisit configuration later as an added feature.

Reply:
ok, I've moved them there 👍🏻